docs/cloud/managing-airbyte-cloud.md

```diff
@@ -87,6 +87,66 @@ Understanding the following limitations will help you better manage Airbyte Clou
 * Shortest sync schedule: Every 60 min
 * Schedule accuracy: +/- 30 min
 
+## View the Sync Log Summary
+The sync log summary shows if a sync succeeded, failed, cancelled, or is currently running.
+ 
+To view the sync log summary:
+1. On the [Airbyte Cloud](http://cloud.airbyte.io/) dashboard, click **Connections**. 
+The Connections page displays.
+2. Click a connection in the list to view its sync history.
+3. Under Sync History, a list of syncs for that connection is displayed.
+The sync status is displayed (Succeeded, Cancelled, Failed, or Running) along with the [summary of the sync log](<https://docs.airbyte.com/cloud/managing-airbyte-cloud#sync-log-summary>).
+4. For more information about a sync, click the sync log dropdown.
+ 
+## Sync Log Summary
+### Succeeded
+The log summary for a successful sync shows the following data.
+
+| Log                            | Description                                                                                                                                             |
+|--------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| x GB (also measured in KB, MB) | Amount of data moved during the sync. If basic normalization is on, the amount of data would not change since normalization occurs in the destination.  |
```

Comment:
@Amruta-Ranade, do you think I should change "x GB (also measured in KB, MB)" to something like "x KB, x MB, or x GB"?

Reply:
Nah I think it's fine as is :)